ocean_lib/assets/test/test_utils.py

```diff
@@ -49,25 +53,42 @@ def test_utilitary_functions_for_trusted_algorithms(publisher_ocean_instance):
         publisher_ocean_instance.config.metadata_cache_uri,
     )
 
-    assert new_publisher_trusted_algorithms is not None
-    assert len(new_publisher_trusted_algorithms) > len(publisher_trusted_algorithms)
+    assert (
+        new_publisher_trusted_algorithms is not None
+    ), "Added a new trusted algorithm failed. The list is empty."
+    assert len(new_publisher_trusted_algorithms) > len(publisher_trusted_algorithms), (
+        "Added a new trusted algorithm "
+        "failed. The length of the new "
+        "trusted algorithms list should "
+        "be bigger than the previous "
+        "one. "
+    )
 
     # add an existing algorithm to publisher_trusted_algorithms list
     new_publisher_trusted_algorithms = add_publisher_trusted_algorithm(
         ddo.did, algorithm_ddo.did, publisher_ocean_instance.config.metadata_cache_uri
     )
     assert new_publisher_trusted_algorithms is not None
     for _, trusted_algorithm in enumerate(publisher_trusted_algorithms):
-        assert trusted_algorithm["did"] == algorithm_ddo.did
+        assert (
+            trusted_algorithm["did"] == algorithm_ddo.did
+        ), "Added a different algorithm besides the existing ones."
     assert len(new_publisher_trusted_algorithms) == len(publisher_trusted_algorithms)
 
     # remove an existing algorithm to publisher_trusted_algorithms list
     new_publisher_trusted_algorithms = remove_publisher_trusted_algorithm(
         ddo.did, algorithm_ddo.did, publisher_ocean_instance.config.metadata_cache_uri
     )
 
-    assert new_publisher_trusted_algorithms is not None
-    assert len(new_publisher_trusted_algorithms) < len(publisher_trusted_algorithms)
+    assert (
+        new_publisher_trusted_algorithms is not None
+    ), "Remove process of a trusted algorithm failed."
+    assert len(new_publisher_trusted_algorithms) < len(publisher_trusted_algorithms), (
+        "Remove process of a trusted "
```

Comment:
Same here, rewrite to be a shorter message. I think the above example works if adjusted to this case.